skimage/transform/tests/test_radon_transform.py

```diff
@@ -66,47 +67,89 @@ def test_iradon_bias_circular_phantom():
     assert roi_err < tol
 
 
-def check_radon_center(shape, circle, dtype, preserve_range):
+def check_radon_center(shape, circle, dtype, preserve_range, center_disp):
```

Comment:
It's funny that, in this file, some test functions have names starting with `check_` (instead of the usual `test_`)...

Reply:
Yes, those `check_` functions are then called by `test_` functions, in a parametrized manner. Some of the `check_` functions are called by different `test_` functions. I suppose they used it to differentiate between the functions doing the job, and the ones selecting the actual configuration to test.
Do we need to maybe change the `check_` functions to something like `_test_`?

Reply:
No, if it works best to leave it, possibly for another PR if the need arises to clean this up.